CMakeLists.txt

```diff
@@ -73,7 +73,12 @@ string(TOLOWER ${ONEDPL_BACKEND} ONEDPL_BACKEND)
 
 if (ONEDPL_ENABLE_SIMD)
     # TODO: remove skipping of Intel® oneAPI DPC++ Compiler once it can handle such options.
-    if (NOT CMAKE_CXX_COMPILER MATCHES ".*dpcpp(-cl)?(.exe)?$")
+    if (CMAKE_CXX_COMPILER MATCHES ".*dpcpp(-cl)?(.exe)?$")
+        execute_process(COMMAND dpcpp --version OUTPUT_VARIABLE VER)
+        string(REGEX MATCH "[0-9][0-9][0-9][0-9]\\.[0-9]\\.[0-9]" DPCPP_VER ${VER})
+        string(REPLACE "." 0 DPCPP_VER ${DPCPP_VER})
```

Comment:
Why do you replace dots?

Reply:
I thought it won't be working this way, but I have tested and now it looks redundant. Simplified.